recipe/deactivate-gfortran.sh

```diff
@@ -1,13 +1,16 @@
-#!/bin/bash
+# shellcheck shell=sh
```

Comment:
Does this then get replaced with `/bin/bash` later? What happens when the compilers are installed and `shellcheck` is not present?

Reply:
The shell should treat this as a comment. I copied this from @mbargull's PR.

Reply:
Yes, this is a comment/directive, not a shebang.  The shebang is ignored anyway because the activate/deactivate scripts are sourced, not exec-ed.